src/v0/documents.ts

```diff
@@ -91,6 +98,21 @@ export class Documents extends ThBaseHandler {
       throw new DocumentsMetaFailed(error.message, { error })
     }
   }
+
+  async send (documentId: string, sendQuery: DocumentsSendQuery): Promise<DocumentsSendResponse> {
+    try {
+      const base = this.uriHelper.generateBaseUri(`/${documentId}/send`)
+      const uri = this.uriHelper.generateUriWithQuery(base)
+
+      const response = await this.http.getClient().post(uri, sendQuery)
+
+      return {
+        success: response.data.success
```

Comment:
nah, that's not it.
This is the complete API response from root level:
```
{
      status: 200,
      msg: 'Success',
      results: [ { success: true }    
}
```

So there it should be what it was before:
`return response.data.results[0]`